src/IssueViz.Security/HotspotsList/HotspotsControl.xaml

```diff
@@ -144,6 +155,58 @@
             <Setter Property="TextWrapping" Value="Wrap"/>
             <Setter Property="TextAlignment" Value="Left"/>
         </Style>
+        <Style TargetType="ContextMenu">
```

Comment:
Is this based on an existing/reference implementation?

Reply:
No, I tweaked it until I thought it looked similar enough.